main/luajit/APKBUILD

```diff
@@ -4,7 +4,7 @@ pkgname=luajit
 _pkgname=LuaJIT
 pkgver=2.1.0_beta3
 _pkgver=${pkgver/_/-}
-pkgrel=4
+pkgrel=5
```

Comment:
why is this needed?

Reply:
Because `abuild` is using `pkg-config --modversion` for package versions and sometimes that value is not legal for use with `apk`.